common/web/gesture-recognizer/src/test/auto/headless/gesturePath.spec.ts

```diff
@@ -19,7 +19,7 @@ describe("GesturePath", function() {
   describe("canary tests: serialization & deserialization", function() {
     it('from fixture', () => {
       let testJSONtext = fs.readFileSync(`${SEGMENT_TEST_JSON_FOLDER}/simple_ne_move.json`);
-      let fullSerializedJSON = JSON.parse(testJSONtext);
+      let fullSerializedJSON = JSON.parse(testJSONtext.toString());
```

Comment:
```suggestion
      let fullSerializedJSON = JSON.parse(testJSONtext);
```

With change above, we don't need `toString()`